packages/camera/camera/lib/src/camera_controller.dart

```diff
@@ -224,26 +224,33 @@ class CameraValue {
 /// To show the camera preview on the screen use a [CameraPreview] widget.
 class CameraController extends ValueNotifier<CameraValue> {
   /// Creates a new camera controller in an uninitialized state.
+  /// Deprecated, use [withSettings].
```

Comment:
Yes, commented as deprecated to avoid breaking changes. According to https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages
...
![image](https://user-images.githubusercontent.com/8981380/234498392-e1a6b4d3-88fa-41a2-b6e4-489dafa62c7c.png)
...